mteb/abstasks/AbsTask.py

```diff
@@ -56,3 +60,15 @@ def evaluate(self, model, split="test"):
         :param split: Which datasplit to be used.
         """
         raise NotImplementedError
+
+    def get_language(self):
+        """ Return the first language of the task.
+        This is not meaningful for multilingual or cross-lingual tasks.
+        Also, a few Nordic tasks although not marked as multilingual, contain multiple languages.
+        For them, the first language will be returned.
```

Comment:
So for those Nordic tasks, it will encode based on the wrong language, correct?

Reply:
It depends on the language definition. For example, `ScalaNbClassification` contains labels `["no", "nb"]`, which means "Norvegian" (an inclusive language name) and "Bokmål Norvegian" (a specific standard of Norvegian), respectively. 
If the model makes a distinction between the different Norvegian standards, then the first code, `no`, is ambiguous for it  (and model has to choose the dialect arbitrarily), but the second one is allright. If the model treats Bokmål and Nynorsk as a single language, then the code `no` is appropriate, and the code `nb` is not adding any new information.

Reply:
I see but for `NordicLangClassification` it would encode using `da` for texts that are in `sv` etc?

Reply:
Yes, it would. But because the goal of the task is to determine the language of the text, we can’t make it much better than choose one language code arbitrarily and pass it with all the inputs.

(and anyway, models like SONAR that are supposed to produce language-agnostic sentence representations aren’t expected to score high in this task, so I think that the choice of this single language code won’t matter much)

Reply:
@avidale. What I would do here instead is pass on the language tag to the model and then let it decide on how to handle it (some models might allow for doing multiple languages at a time).